middleware/multer.js

```diff
@@ -0,0 +1,23 @@
+const multer = require('multer');
+const path = require('path');
+const uploadDir = path.join(process.cwd(), 'temp');
```

Comment:
folder "temp" na starcie nie będzie istaniał bo jest pusty a git nie przechowuje folderów. To samo wydarzyło by się z public i public/avatars gdyby nie to że masz tam zdjęcie.

Przy starcie servera trzeba sprawdzić czy foldery "temp", "public", "public/avatars" istnieją i jeśli nie to je utworzyć. Możesz to zrobić za pomcą biblioteki fs (on jest już w node nie musisz instalować)

```
const fs = require('fs');

const createDirIfNotExists = dir =>
  !fs.existsSync(dir) ? fs.mkdirSync(dir) : undefined;

createDirIfNotExists('temp')
createDirIfNotExists('public')
createDirIfNotExists('public/avatars')
```